### ISSUE TITLE
NTR: Caffeine / halothane in vitro contracture test positive (Lab: neurophysiology: NCV/EMG)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9050]
New term request Caffeine / halothane in vitro contracture test positive (segal_040106182457): IVCT; CHCT  UMLS: 2006084

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000149
- label: Reduced in vitro contracture test threshold
- parents: [Abnormal muscle physiology (HP:0011804)](https://hpo.jax.org/app/browse/term/HP:0011804)
- pmid: PMID:1833602
- def: The in vitro contracture test determines the sensitivity of freshly obtained skeletal muscle specimens to caffeine or halothane applied to a bathing solution. Muscles from persons susceptible to malignant hyperthermia have lower contracture thresholds for these agents than do normal muscle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9050
- synonyms: Caffeine in vitro contracture test positive(NONE)|Halothane in vitro contracture test positive(NONE)
